pages/plugins/decorators.rst

```diff
@@ -0,0 +1,63 @@
+.. _decorators_api:
+
+**********
+Decorators
+**********
+
+:ref:`decorators` can be used to transform a message field value during searches. Multiple decorators can be applied at the same time, but you cannot make any assumptions about their order, as that is user defined.
```

Comment:
Are stacked decorators receiving the transformed message field content from previous decorators or the pristine message field content?